src/js/main.js

```diff
@@ -18,24 +18,35 @@ let app;
 let loadedConfig;
 
 // Retrieve the config by API key.
-let appName = getAppName();
-if (!appName) {
+let appName = getParam('appName');
+let jwKey = getParam('jwKey');
+
+if (appName) {
+    ConfigLoader.getConfig(appName)
+        .then((config) => {
+            // Set the player key.
+            jwplayer.key = config.key;
```

Comment:
Currently the configurations returned by the Showcase API will return `null` for keys in the config, doing it this way will allow us to also override those keys from the URL.